docs/root/start/quick-start/run-envoy.rst

```diff
@@ -104,6 +104,51 @@ The ``-c`` or ``--config-path`` flag tells Envoy the path to its initial configu
                    -c /envoy-custom.yaml
          ...
 
+   .. tab:: Windows Service
+
+      .. note::
+
+         This feature is still in Experimental state.
+
+      You can start Envoy as Windows Service that is managed under `Windows Service Control Manager <https://docs.microsoft.com/en-us/windows/win32/services/using-services/>`_. 
+
+      First, you need to create the service. Assuming you have a custom configuration in the current directory named ``envoy-custom.yaml``. After you create the service you
+      can start it.
+      
+      From an **administrator** prompt run the following commands (note that you need replace C:\EnvoyProxy\ with the path to the envoy.exe binary):
+
+      .. substitution-code-block:: console
+
+         > sc create EnvoyProxy binpath="C:\EnvoyProxy\envoy.exe --config-path envoy-demo.yaml"
```

Comment:
Rather than reopening the "resolved" conversation, we should start a new one... what is the path to envoy-demo.yaml? Similar question as before.

Secondly I see the link below, but do we want to suggest `start=auto` param, since that's what most folks would expect to be the "default" behavior when setting up envoy-as-a-service?

Third, I'd failed to consider before when we decided on leveraging sc create... in another implementation, I'd discovered the following bare minimum service dependencies... Tcpip and Afd need to already be running. Do we want to propose `depend=Tcpip/Afd` as a starting point, and iterate on other dependencies we discover?

Reply:
good call, added them https://github.com/envoyproxy/envoy/pull/14352/commits/44ce57b0a84e1c80792ff756a5cf530a930eb15d